tests/Fixtures/Integration/misc/meta_insert_64566_tokens.test

```diff
@@ -1,6 +1,10 @@
 --TEST--
 Test of super huge file that would require 64566 tokens to be inserted! Basically, this tests Tokens::insertSlices, without which this test would take few hours to execute.
+
+Test is disabled for MacOS because it regularly fails with timeout (see: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7141)
 --RULESET--
 {
     "whitespace_after_comma_in_array": true
 }
+--REQUIREMENTS--
+{"os":["Linux","Windows"]}
```

Comment:
> I actually wonedr about whole idea here.
> I understand the need to disable the test on MacOS.
> but I wonder if anyone would ever use this `requirements['os']` other than this single performance test, that is failing due to CI problems and not due to MacOS itself.

ie, if we ever use this logic only for that one single test, maybe it's showing that this is _not_ an integration test and we should handle performance test differently.

I'm not concluding on decision or direction, only wondering.

Reply:
We have many unit tests that use `@require OS ...`, this PR brings the same functionality to integration tests. It can help us with failing test, but also allow to fine-tune integration tests.

Reply:
We can bring this test back on MacOS when Github Action is fixed. Or we can revert changes in this file and just bump time limit for large tests, as another workaround for our CI problem.

Reply:
Today I had to restart MacOS job NINE times to make CI green, so I am going to merge this now to get rid of this problem. This is not crucial test and I believe it's totally OK if we run it only on other OSes. We'll try to bring this test back when OPcache on MacOS is fixed.